Description: set minimum vt to 7
 maybe this will fix switching vt when installing packages and integration with ubiquity-dm?
Author: Jonathan Riddell
Origin: kubuntu
Last-Update: 2015-03-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: sddm-0.11.0/CMakeLists.txt
===================================================================
--- sddm-0.11.0.orig/CMakeLists.txt
+++ sddm-0.11.0/CMakeLists.txt
@@ -121,7 +121,7 @@ if(SYSTEMD_FOUND)
         string(REGEX REPLACE "[ \t\n]+" "" SYSTEMD_SYSTEM_UNIT_DIR ${SYSTEMD_SYSTEM_UNIT_DIR})
     endif()
 
-    set(MINIMUM_VT 1)
+    set(MINIMUM_VT 7)
     set(HALT_COMMAND "/usr/bin/systemctl poweroff")
     set(REBOOT_COMMAND "/usr/bin/systemctl reboot")
 else()
